src/views/PeopleSearch/components/PeopleSearchResult/index.js

```diff
@@ -119,15 +119,23 @@ export default class PeopleSearchResult extends Component {
                   placeholderColor="#eeeeee"
                 />
               </Grid>
-              <Grid item xs={6}>
-                <Typography variant="h5">{fullName}</Typography>
-                <Typography variant="body2">{nickname}</Typography>
-                <Typography variant="body2">
-                  {personClassJobTitle}
-                  {Person.Type === 'Alum' ? Person.PreferredClassYear : null}
+              <Grid item xs={8}>
+                <Typography variant="h5">
+                  {Person.FirstName} {nickname} {Person.LastName}
                 </Typography>
-                <Typography variant="body2">{Person.Email}</Typography>
-                <Typography variant="body2">{personMailLocation}</Typography>
+                <div style={{ color: '#666666' }}>
+                  <Typography variant="body2">
+                    {personClassJobTitle ?? Person.Type}
+                    {Person.Type === 'Alum' ? ' ' + Person.PreferredClassYear : null}
+                  </Typography>
+                  <Typography variant="body2">
+                    {Person.Major1Description}
+                    {Person.Major2Description ? `, ${Person.Major2Description}` : null}
+                    {Person.Major3Description ? `, ${Person.Major3Description}` : null}
+                  </Typography>
+                  <Typography variant="body2">{Person.Email}</Typography>
+                  <Typography variant="body2">{personMailLocation}</Typography>
+                </div>
```

Comment:
I like the effect of having grayer text! There is a standard MUI way to achieve this that is used elsewhere in the site - by setting the `color` prop of `Typography` to `textSecondary`. That has the disadvantage of having to be repeated for every `Typography` (unless you create a custom `SecondaryText` component).
```javascript
const SecondaryText = ({children, otherProps}) => <Typography variant="body2" color="textSecondary" {...otherProps}>{children}</Typography>
...
<SecondaryText>{Person.Email}</SecondaryText>
<SecondaryText>{personMailLocation</SecondaryText>
```

If you do choose to keep the enclosing `div` with a color that's inherited, then you should at least move the styling to a class in the `scss` file. That way the style is declared only once in the class, and each div uses that className. Then, if we ever need to change the style, we only have to change the class, not every individually styled div.